magda-registry-aspects/currency.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "http://json-schema.org/hyper-schema#",
+    "title": "Currency",
+    "description": "Describes how up-to-date the dataset is",
+    "type": "object",
+    "properties": {
+        "status": {
+            "description": "the status of the this dataset",
+            "type": "string",
+            "enum": ["CURRENT", "SUPERSEDED", "RETIRED"]
+        },
+        "supersededBy": {
+            "description": "What dataset has it been superseded by?",
```

Comment:
This should be a plural (i.e. "What dataset(s) has it been superseded by?")

Reply:
Will update~ 👍